task.sql

```diff
@@ -2,10 +2,17 @@
 USE ShopDB; 
 
 -- Some data should be created outside the transaction (here)
+INSERT INTO Orders (CustomerID, Date)
+VALUES (1, "2024-01-01");
 
--- Start the transaction 
-START TRANSACTION; 
+-- Start the transaction
+START TRANSACTION;
 
--- And some data should be created inside the transaction 
+INSERT INTO OrderItems (OrderID, ProductID, Count)
+VALUES (1, 1, 1);
 
-COMMIT; 
\ No newline at end of file
+UPDATE Products SET WarehouseAmount = WarehouseAmount - 1
+WHERE ID = 1;
+-- And some data should be created inside the transaction
+
+COMMIT;
```

Comment:
![image](https://github.com/user-attachments/assets/a2ffdace-e42a-4ebd-86fb-b76c99c059f1)
Add Eof(End-of-file), need to add blank line at the end